scripts/scripts.js

```diff
@@ -709,6 +710,20 @@ export function addPagingWidget(
   div.appendChild(nav);
 }
 
+export async function fetchTagsOrCategories(ids = [], sheet = 'tags', type = '', locale = 'en') {
+  const placeholders = await fetchPlaceholders(locale);
+  await loadScript('/ext-libs/jslinq/jslinq.min.js');
```

Comment:
I think we should load only if needed and first check if its already present in `window`